docs/site/migration/auth/example.md

```diff
@@ -636,6 +559,56 @@ $ lb4 repository
 ? Please select the repository base class DefaultCrudRepository (Legacy juggler bridge)
 ```
 
+Create relations:
+
+- `Project` belongsTo `User`
+
+```sh
+lb4 relation
+? Please select the relation type belongsTo
+? Please select source model Project
+? Please select target model User
+? Foreign key name to define on the source model ownerId
+? Relation name owner
+? Allow Project queries to include data from related User instances? Yes
+```
+
+{% include tip.html content="
+Please delete the generated controller file `src/controllers/project-user.controller.ts` to keep the exposed endpoints clean.
+" %}
+
+- `User` hasOne `UserCredentials`
```

Comment:
Similar comment ... We've defined the relation when changing the user.model.ts

Reply:
removed as https://github.com/strongloop/loopback-next/pull/4855#discussion_r391254286